app/routes/administrative-units/administrative-unit/governing-bodies/governing-body/mandatory/edit.js

```diff
@@ -0,0 +1,52 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class AdministrativeUnitsAdministrativeUnitGoverningBodiesGoverningBodyMandatoryEditRoute extends Route {
+  @service store;
+
+  async model({ mandatoryId: mandatoryId }) {
+    let adminUnitGovBodyTemps = this.modelFor(
+      'administrative-units.administrative-unit.governing-bodies'
+    );
+
+    let { governingBodyId: govBodyTempId } = this.paramsFor(
+      'administrative-units.administrative-unit.governing-bodies.governing-body'
+    );
+
+    let govBodyTemp = await this.store.findRecord(
```

Comment:
I think you can reuse the record that's fetched in that route already?

Also I think the variable naming can be improved or is this test code? :smile: 

I think something like this could work?
```js
    let { govBodyTimeSpec } = this.modelFor(
      'administrative-units.administrative-unit.governing-bodies.governing-body'
    );
```

Reply:
I thought this, but there's much stuff there that won't be needed in the edit. Only the governing body period is needed.

Reply:
Yea, but it will be fetched anyway, since it's an ancestor route. Fetching the record here again would only be needed if we need _extra_ data that isn't fetched there.